drivers/adc/adc_stm32.c

```diff
@@ -476,6 +480,20 @@ static int adc_stm32_check_acq_time(uint16_t acq_time)
 	return -EINVAL;
 }
 
+/*
+ * Enable internal voltage reference source
+ */
+static void adc_stm32_set_common_path(const struct device *dev)
+{
+	const struct adc_stm32_cfg *config =
+		(const struct adc_stm32_cfg *)dev->config;
+	ADC_TypeDef *adc = (ADC_TypeDef *)config->base;
+	(void) adc; /* Avoid 'unused variable' warning for some families */
```

Comment:
Given the code below, I wonder in which conditions you can get this 'unused variable' warning,
can you clarify ?

Reply:
@erwango For example, modules\hal\stm32\stm32cube\stm32f0xx\drivers\include\stm32f0xx_ll_adc.h:
```
#define __LL_ADC_COMMON_INSTANCE(__ADCx__)                                     \
  (ADC1_COMMON)
```